### PR TITLE
bdist_conda edits per meeting

### PR DESCRIPTION
**Pull request by kerrywatson1**
@asmeurer @ilanschnell -- per work session today. Please review and merge. @electronwill

### CODE REVIEW

**Comment by ilanschnell** [comment]
Looks good to me.

**Comment by kerrywatson1** [comment]
@asmeurer please merge.

File: docs/source/bdist_conda.rst [hunk 1]
```diff
@@ -1,91 +1,141 @@
...
+
+Build number
+-------------
+
+Set the build number. Defaults to the ``conda_buildnum`` passed to ``setup()``, or 0. Overrides any 
+``conda_buildnum`` passed to ``setup()``.
+
+.. code::
+
+   --buildnum
```
Comment (asmeurer) [inline docs/source/bdist_conda.rst]: There should be an argument, like `--buildnum 1`.

File: docs/source/bdist_conda.rst [hunk 2]
```diff
@@ -1,91 +1,141 @@
...
 
-- ``conda_features``: List of features for the package. See the
-  :ref:`features` section of the conda build documentation for more
-  information about features in conda.
+.. code::
 
-- ``conda_track_features``: List of features that this package should track
-  (enable when installed).  See the :ref:`features` section of the conda build
-  documentation for more information about features in conda.
+   Track features
```
Comment (asmeurer) [inline docs/source/bdist_conda.rst]: `conda_track_features`

**Comment by asmeurer** [comment]
The formatting of these examples seems weird. If we're not going to show an example (like `conda_buildnum=1` maybe we should format the name of the item itself as the header somehow, or inline with the text.

**Comment by kerrywatson1** [comment]
Okay, @asmeurer Ilan and I edited the file. Please push and have a great weekend!

**Comment by kerrywatson1** [comment]
@asmeurer @ilanschnell finally ready w/Aaron's and Ilan's edits, please merge this puppy.

File: docs/source/bdist_conda.rst [hunk 3]
```diff
@@ -1,91 +1,156 @@
...
 
-Options that can be passed to ``setup()`` (must include
-``distclass=CondaDistribution``):
+Command line tests to run. Default is True, which runs ``command --help`` for each command in the 
+``console_scripts`` and ``gui_scripts entry_points``. Also allowed are False, which doesn’t run any 
+command tests, or a list of command tests to run.
+
+.. code::
+
+   conda_command_tests =False
```
Comment (asmeurer) [inline docs/source/bdist_conda.rst]: There's an extra space here.

File: docs/source/bdist_conda.rst [hunk 4]
```diff
@@ -1,91 +1,156 @@
...
+   from distutils.core import setup, Extension
+   import distutils.command.bdist_conda
 
-Notes
-=====
+   setup(
+      name="foo",
+      version="1.0",
+      distclass=distutils.command.bdist_conda.CondaDistribution,
+      conda_buildnum=1,
```
Comment (asmeurer) [inline docs/source/bdist_conda.rst]: There's no closing parenthesis here.

**Comment by kerrywatson1** [comment]
With @asmeurer edits, please merge.

**Pull request merged**